src/components/general/NavLink/NavLink.scss

```diff
@@ -19,7 +19,7 @@
   &:hover,
   &-active {
     @media (hover: hover) {
-      background: rgba(51, 150, 255, 0.08);
+      background: var(--accent-hover);
```

Comment:
The variables might needed to be updated in the future but for now, I've set custom color for the dark theme and started using this variable on every hover background.